Chat/assignment-chat/server-chat/src/main/java/ru/assignment/model/ChatModelListener.java

```diff
@@ -0,0 +1,12 @@
+package ru.assignment.model;
+
+import ru.assignment.message.ChatMessage;
+
+/**
+ * Created by Андрей on 19.02.2015.
+ */
+public interface ChatModelListener {
+    void sendMessageToClient(ChatMessage chatMessage);
```

Comment:
Please, use event like method names:
listener#onNewMessage()
listener#onSomething()
listener#onDisconnectReceived

Don't use action-like methods
listener#doSomething
listener#sendMessage 

Because not always listener will actually send meesage, it can also log it.